src/components/EventCard/EventCard.js

```diff
@@ -100,46 +103,58 @@ function EventCard({
 	rsvpLink,
 	imgFile,
 	disabled,
-	classes
+	classes,
 }) {
 	const [isHover, setIsHover] = useState(false);
 	const dateStr = dayjs(date).calendar();
-	const isWithin12Hours = dayjs.utc()
+	const isWithin12Hours = dayjs
+		.utc()
 		.isBetween(
-			dayjs(date).utc().subtract(12, 'hours'),
-			dayjs(date).utc().add(12, 'hours')
+			dayjs(date).utc().subtract(12, "hours"),
+			dayjs(date).utc().add(12, "hours")
 		);
 	return (
 		<Card
 			raised
 			elevation={isHover ? 11 : 6}
 			className={classNames(classes.container, {
 				[classes.greyOverlay]: disabled,
-				[classes.hoverTranslation]: isHover
+				[classes.hoverTranslation]: isHover,
 			})}
 			onMouseEnter={() => setIsHover(true)}
 			onMouseLeave={() => setIsHover(false)}
 		>
 			{/* Empty string added as child to squelch CardMedia warning */}
 			<CardMedia classes={{ root: classes.banner }}>
-				<GatsbyImage image={imgFile.childImageSharp.gatsbyImageData} />
+				<GatsbyImage
+					image={imgFile.childImageSharp.gatsbyImageData}
+					imgStyle={{ borderRadius: classes.gatsbyBorderRadius }} //For iOS img borders
```

Comment:
One reason why it's not working could be because you're adding the style wrong, since imgStyle expects a javascript object, you could try:
`imgStyle={classes.gatsbyBorderRadius}`

Reply:
This doesn't work unfortunately, causes some issues with react-dom development.js in node_modules

Will continue to look for a solution